lib/init.coffee

```diff
@@ -6,13 +6,13 @@ module.exports =
     executablePath:
       title: 'Executable Path'
       type: 'string'
-      default: 'pep257'
-      description: "Path to executable pep257 cmd."
+      default: 'pydocstyle'
```

Comment:
Good catch, I hadn't even thought about it. If this were a heavily used plugin, I think it'd be worth the migration effort to implement a fallback. But it's not that heavily used, so I don't think it's worth it... a note on the Readme alerting folks about the switch should suffice.